functions/4dn-status/src/qa_utils_subset.py

```diff
@@ -0,0 +1,133 @@
+"""
+Functionality borrowed from dcicutils.qa_utils.
```

Comment:
Heh. Partly I need to port some new functionality to there. But also, I feel like bringing in dcicutils brings in a zillion libraries and that that's bad for a lambda trying to be small and fast.  It's a policy decision we should decide together; it's been on my list to discuss this with you. So let's talk about it and then maybe fix this.

Reply:
I'll make this a separate task since it could destabilize this patch to just toss it in last minute. It will also let me get some timing information if we put in one then change to the other.